speed-run/mcp/server.py

```
@@ -0,0 +1,337 @@
+                                "status": "ok",
+                                "url": CEREBRAS_URL,
+                                "model": CEREBRAS_MODEL,
+                                "available_models": models,
+                            }
+                        else:
+                            result = {
+                                "status": "error",
+                                "error": f"HTTP {response.status_code}",
+                                "url": CEREBRAS_URL,
+                            }
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)